src/components/ux-tabarea/ux-tabarea.steps.js

```diff
@@ -26,5 +26,5 @@ module.exports = function () {
 	this.Given(/^I have loaded component "([^"]*)" with use case "([^"]*)"$/,
 		function (componentName, useCase, callback) {
 			this.client.loadComponentWithUseCase(componentName, useCase, callback);
-	});
+		});
```

Comment:
If you line the curly braces (}) you should also align the round brackets ie split this line into to.